js_task2/src/scss/modal.scss

```diff
@@ -0,0 +1,66 @@
+.modal-window {
+  position: fixed;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+
+  background-color: rgba(128, 128, 128, 0.555);
+
+  form {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    padding: 24px;
+    min-width: 350px;
```

Comment:
same as 3line in general.scss

it's interesting that you have a form with min-width 350px inside a container with min-width 320px, so it won't fit it

Reply:
deleted this